Modules/QuestieQuestLog.lua

```diff
@@ -0,0 +1,239 @@
+---@class QuestieQuestLog
+local QuestieQuestLog = QuestieLoader:CreateModule("QuestieQuestLog")
+
+local elvEnabled = IsAddOnLoaded("ElvUI")
+
+function QuestieQuestLog:IsElvSkinningEnabled(E)
```

Comment:
Can you explain what `E` is in this case? :)

Reply:
Do you mean by adding a docblock to the function explaining what E is? I don't know LUA very well so I am unaware of what docblock annotations are available :>